crates/cli/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "gevulot-cli"
```

Comment:
For the name by exp I know I'm not very good, so I take what it's done in other BC. But you can define any name, I don't really have any good idea.
I agree, it's better if the tool have its own repo because update and release rhythm and their consequences are not the same for the node and the client.
More generally, I agree with the Rust idiomatic's way to define crate:  define small crate. The drawback is that it add complexity when you have a lot of crate not published in crates.io because you've to manage all the specific commit for all crates you use. It can become tedious when all crates move fast before the first release.